samples/spring-reliable-timers/src/main/java/com/example/actions/OrderAction.java

```diff
@@ -75,20 +78,15 @@ public Effect<Order> placeOrder(@RequestBody OrderRequest orderRequest) {
   @PostMapping("/expire/{orderId}")
   public Effect<String> expire(@PathVariable String orderId) {
     logger.info("Expiring order '{}'", orderId);
-
-    Predicate<StatusRuntimeException> validateErrorCodes = exception -> {
-      Status.Code code = exception.getStatus().getCode();
-      return code == Status.Code.NOT_FOUND || code == Status.Code.INVALID_ARGUMENT;
-    };
-
     var cancelRequest = kalixClient.post("/order/"+orderId+"/cancel", "", String.class);
 
     CompletionStage<String> reply =
         cancelRequest
             .execute() // <1>
             .thenApply(cancelled -> "Ok") // <2>
             .exceptionally(e -> { // <3>
-                  if (e instanceof StatusRuntimeException && validateErrorCodes.test((StatusRuntimeException) e)) {
+                  if (e.getCause() instanceof DeferredCallResponseException ere &&
```

Comment:
ere?

Reply:
the exception had a different name before.. 😂 will update. tks